### PR TITLE
Reformat python files of mobly tests/lib.

### PR DESCRIPTION
**Pull request by eric100lin**
Make the indentation style of mobly consistent with that of google pyformat.

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/700)

### CODE REVIEW

File: tests/lib/integration2_test.py [hunk 1]
```diff
@@ -14,14 +14,15 @@
 
 from mobly import test_runner
 
-from tests.lib import integration_test
```
Comment (xpconanfan) [inline tests/lib/integration2_test.py:17]: why is this line changed?
we should strictly do indentation change first, then fix tidbits

**Review by xpconanfan — CHANGES_REQUESTED** [review]

File: tests/lib/integration2_test.py (hunk 1 above)
Comment (eric100lin) [reply tests/lib/integration2_test.py:17]: This line is accidentally changed. I reverted it.

**Comment by eric100lin** [comment]
Rework and create another new PR.
